src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.csproj

```diff
@@ -6,17 +6,17 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Language.TextAnalytics</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services Language APIs.</Description>
-    <Version>2.8.0-preview</Version>
+    <Version>2.8.0</Version>
```

Comment:
As there is a breaking change I believe version should bump.

Reply:
If you intend to land on a stable version, please bump this to `3.0.0`

Reply:
This might also need a bump in the `AssemblyInfo.cs` file